ansible_base/resource_registry/signals/handlers.py

```diff
@@ -18,10 +15,8 @@ def get_resource_models():
     return resource_models
 
 
-@receiver(post_delete)
+# @receiver(post_delete)
```

Comment:
Delete comment. Moving the connection to the app's `.ready` call is the right thing to do.